themes/bootstrap3/templates/devtools/language.phtml

```diff
@@ -15,8 +15,8 @@
     ?>
     <tr>
       <td><?=$this->escapeHtml($langCode . ' (' . $diffs['name'] . ')')?></td>
-      <td><?=$missingCount ?><?php if($missingCount > 0): ?> (<a href="#" onclick="return diffManager.showMissing('<?=$langCode ?>')"><?=$this->transEsc('show') ?></a>)<?php endif; ?></td>
-      <td><?=$extraCount ?><?php if($extraCount > 0): ?> (<a href="#" onclick="return diffManager.showExtra('<?=$langCode ?>')"><?=$this->transEsc('show') ?></a>)<?php endif; ?></td>
+      <td><?=$missingCount ?><?php if($missingCount > 0): ?> (<a class="missing_count" data-langcode="<?=$langCode?>" href="#"><?=$this->transEsc('show') ?></a>)<?php endif; ?></td>
+      <td><?=$extraCount ?><?php if($extraCount > 0): ?> (<a class="extra_count" data-langcode="<?=$langCode?>" href="#"><?=$this->transEsc('show') ?></a>)<?php endif; ?></td>
```

Comment:
These should probably be turned into buttons instead of href-free anchors.

Reply:
I will do that on #1639